test/unit/virtual_include/BUILD

```diff
@@ -0,0 +1,70 @@
+# Copyright 2023 Ericsson AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# cc_binary for simple C++ tests
+load(
+    "@rules_cc//cc:defs.bzl",
+    "cc_binary",
+    "cc_library",
+)
+
+# codechecker rules
+load(
+    "@bazel_codechecker//src:codechecker.bzl",
+    "codechecker_test",
+)
+
+# Prototype for CodeChecker analyze --file
+# NOTE: CodeChecker analyze --file --ctu does not work
+load(
+    "@bazel_codechecker//src:code_checker.bzl",
+    "code_checker_test",
+)
+
+# Test for strip_include_prefix
+cc_library(
+    name = "test_inc",
+    hdrs = glob(["inc/*.h"]),
+    # NOTE: the following is for test purpose only
+    # NOTE: use includes instead of strip_include_prefix
+    strip_include_prefix = "inc",
```

Comment:
Let's talk about this -- is it terribly troublesome to do the normal way? I'm fine leaving this as-is, just want to understand.